general/releases/2.3/2.3.5.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Moodle 2.3.5
+tags:
+  - Release notes
+  - Moodle 2.3
+sidebar_position: 5
+moodleVersion: 2.3.5
+---
+<p class="note">Because of a serious regression in the 2.3.5 release, [Moodle 2.3.6](/general/releases/2.3/2.3.6) was release on Mon 18 March, 2013.</p>
```

Comment:
```suggestion

:::note

Because of a serious regression in the 2.3.5 release, [Moodle 2.3.6](/general/releases/2.3/2.3.6) was released on Mon 18 March, 2013.

:::
```